pkg/config/shortnames/util.go

```diff
@@ -0,0 +1,150 @@
+package shortnames
+
+import (
+	"bytes"
+	"os"
+	"path/filepath"
+
+	"github.com/containers/image/v5/pkg/sysregistriesv2"
+
+	"github.com/BurntSushi/toml"
+	"github.com/containers/storage/pkg/unshare"
+	"github.com/pkg/errors"
+)
+
+//// ALGORITHMIC UTILITY ////
+
+// linear search returning location of element in a slice (returns -1 if not found)
+func find(a []string, x string) int {
```

Comment:
`findStringInSlice`? “find” is awfully generic even for the current amount of code.

Anyway, the only caller immediately follows with `removeIndex`, so the two could be combined into a `stringSliceWithoutValue` or something similar.